src/d3a/d3a_core/export.py

```diff
@@ -217,8 +217,8 @@ def _export_area_clearing_rate(self, area, directory, file_suffix, is_first):
                 for market in area.past_markets:
                     for time, clearing in (
                             bid_offer_matcher.matcher.match_algorithm.state.clearing.items()):
-                        if market.time_slot > time:
-                            row = (market.time_slot, time, clearing)
+                        if market.time_slot_str > time:
+                            row = (market.time_slot_str, time, clearing)
```

Comment:
This does not seem right. I wonder how you can compare a string and a time.

Reply:
Thanks for noticing, it was changed in between commits. Corrected it now: https://github.com/gridsingularity/d3a/pull/1195/commits/f362d6a5e707ffbdf942f24a49376a3fe634cc98